src/Model/Table/LazyTableTrait.php

```diff
@@ -45,7 +45,7 @@ public function ensureTables(array $fixtures)
             if ($class === false) {
                 throw new \RuntimeException("Unknown fixture '$name'.");
             }
-            $fixture = new $class();
+            $fixture = new $class($this->connection()->configName());
```

Comment:
This might indeed need a modification of the constructor of TestFixture. There is currently no way to set the connection to use by a Fixture except in the property field (and the change to the fixtures of the plugin might cause strict notices).